src/http/fetcher.ts

```diff
@@ -181,6 +163,11 @@ export class Fetcher {
     for (const key of Object.keys(pathParams)) {
       url = url.replace(`{${key}}`, encodeURIComponent(pathParams[key]));
```

Comment:
I forgot to skip this when the value is undefined

Reply:
I did that, sourceKey is injected automatically if undefined in path.